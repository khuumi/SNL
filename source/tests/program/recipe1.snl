recipe some_recipe:
start some_stage:
5
done
done