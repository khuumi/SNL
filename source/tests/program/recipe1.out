recipes = [{ rname = "some_recipe"
   formals = []
   body = [{ sname = "some_stage"
   is_start = true
   body = [Expr (Int 5)]}
]}
],
stages = []